GitUp/Application/AppDelegate.m

```diff
@@ -560,7 +560,7 @@ - (IBAction)newRepository:(id)sender {
   if ([savePanel runModal] == NSFileHandlingPanelOKButton) {
     NSString* path = savePanel.URL.path;
     NSError* error;
-    if (![[NSFileManager defaultManager] fileExistsAtPath:path] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
+    if (![[NSFileManager defaultManager] fileExistsAtPath:path followSymlink:NO] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
```

Comment:
Per your quote, it does, but not the last one. So this is a different behavior than your new API which doesn't follow any symlink, right?